stepfunction/s3.py

```diff
@@ -0,0 +1,21 @@
+def get_activity_arn(record, client):
```

Comment:
why `s3` for the filename? Those functions are related to stepfunctions, not aws storage.